app/backend/src/couchers/migrations/versions/a0d7c8af5402_modify_the_database.py

```diff
@@ -0,0 +1,270 @@
+"""Modify the database
+
+Revision ID: a0d7c8af5402
+Revises: 34cb1d27d207
+Create Date: 2020-12-17 16:43:54.221781
+
+"""
+import geoalchemy2
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a0d7c8af5402"
+down_revision = "34cb1d27d207"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("SET CONSTRAINTS ALL DEFERRED")
+    op.create_table(
+        "clusters",
+        sa.Column("id", sa.BigInteger(), nullable=False),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("main_page_id", sa.BigInteger(), nullable=False),
+        sa.Column("created", sa.DateTime(timezone=True), server_default=sa.text("now()"), nullable=False),
+        # sa.ForeignKeyConstraint(["main_page_id"], ["pages.id"], name=op.f("fk_clusters_main_page_id_pages")),
```

Comment:
Why no FK?

Reply:
It gave an error because the pages table does not exist yet. So I moved the creation of this foreign key to just after the creation of the pages table.

Reply:
The same for a foreign key to threads in pages.